docs/Gemfile.lock

```diff
@@ -30,18 +30,19 @@ GEM
       redcarpet (~> 3.1)
       safe_yaml (~> 1.0)
       toml (~> 0.1.0)
-    jekyll-coffeescript (1.0.1)
+    jekyll-coffeescript (1.0.2)
       coffee-script (~> 2.2)
+      coffee-script-source (~> 1.11.1)
     jekyll-gist (1.4.0)
-      octokit (~> 4.3.0)
+      octokit (~> 4.2)
```

Comment:
There was never an explicit version downgrade in `jekyll-gist`, they always had the `4.3.0` `octokit` dependency since it was added.

Similarly, since we added the `jekyll-gist`/`octokit` dependency the version of `octokit` was set to `4.3.0`.

In conclusion, it doesn't appear like it's related to a bug fix. We just always relied on a higher version than `jekyll-gist` did, and it worked out fine.